kebechet/kebechet_runners.py

```diff
@@ -192,9 +192,15 @@ def run(
             )
             instance.run(**manager_configuration)
         except Exception as exc:  # noqa F841
-            _create_issue_from_exception(
-                manager_name=manager_name, ogr_service=ogr_service, slug=slug, exc=exc
-            )
+            if (keb_config := config.config.get("kebechet")) and keb_config.get(
```

Comment:
Not very happy to see this operator used - some linters banned it and it is not that straightforward to read.

Reply:
Only this comment - otherwise changes look good to me. If you want to keep the walrus operator, feel free to merge this.